tests/unit/internal-session-test.js

```diff
@@ -6,7 +6,7 @@ import { expect } from 'chai';
 import sinon from 'sinon';
 import InternalSession from 'ember-simple-auth/internal-session';
 import EphemeralStore from 'ember-simple-auth/session-stores/ephemeral';
-import Authenticator from 'ember-simple-auth/authenticators/base';
+import Authenticator from 'ember-simple-auth/authenticators/test';
```

Comment:
Actually it isn't, just thought it would be better to test on an authenticator that resolves by default. It doesn't affect the tests anyway as most of it has been stubbed so I changed it back